<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImagingPlannerOptions</class>
 <widget class="QWidget" name="ImagingPlannerOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>200</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QCheckBox" name="CenterOnSkyMapCB">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Slew to selected objects on the SkyMap.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Center selected object on SkyMap</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="IndependentWindowCB">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Make the ImagingPlannerTool an independent window. Would take effect next restart.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Independent Window</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="StartSolvingImmediatelyCB">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start solving immediately after taking a screenshot, instead of waiting for the Solve button to be pressed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Start Solving Immediately</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
